From 8b5abaf80342d448ea31a534a2c7539214b96971 Mon Sep 17 00:00:00 2001
From: J. Bruce Fields <bfields@redhat.com>
Date: Mon, 23 Jun 2014 21:49:35 -0400
Subject: [fs] nfsd:Define op_iattr for nfsd4_open instead using macro

Message-id: <1403560314-32525-34-git-send-email-bfields@redhat.com>
Patchwork-id: 84458
O-Subject: [Patch RHEL7 033/172] Define op_iattr for nfsd4_open instead using macro
Bugzilla: 1111719
RH-Acked-by: Steve Dickson <SteveD@redhat.com>

From: Kinglong Mee <kinglongmee@gmail.com>

Signed-off-by: Kinglong Mee <kinglongmee@gmail.com>
Signed-off-by: J. Bruce Fields <bfields@redhat.com>
Upstream: 3ff69309fed8ac3755864addfa064b51abfcde06
Bugzilla: 1111719
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/fs/nfsd/xdr4.h b/fs/nfsd/xdr4.h
index 916a407..d278a0d 100644
--- a/fs/nfsd/xdr4.h
+++ b/fs/nfsd/xdr4.h
@@ -228,7 +228,7 @@ struct nfsd4_open {
  u32  op_create;          /* request */
  u32  op_createmode;      /* request */
  u32  op_bmval[3];        /* request */
- struct iattr iattr;              /* UNCHECKED4, GUARDED4, EXCLUSIVE4_1 */
+ struct iattr op_iattr;           /* UNCHECKED4, GUARDED4, EXCLUSIVE4_1 */
  nfs4_verifier op_verf __attribute__((aligned(32)));
          /* EXCLUSIVE4 */
  clientid_t op_clientid;        /* request */
@@ -250,7 +250,6 @@ struct nfsd4_open {
  struct nfs4_acl *op_acl;
  struct xdr_netobj op_label;
 };
-#define op_iattr iattr
 
 struct nfsd4_open_confirm {
  stateid_t oc_req_stateid  /* request */;
-- 
1.7.1